From 4444b7c5ca7a8cd3e018f4c421b0737159be9ab8 Mon Sep 17 00:00:00 2001
From: Ilikara <3435193369@qq.com>
Date: Sun, 7 Dec 2025 21:09:34 +0800
Subject: [PATCH 345/345] AOSCOS: nvme-pci: add Wodposit NVMe SSD to
 nvme_id_table and apply NVME_QUIRK_NO_SECONDARY_TEMP_THRESH

Signed-off-by: Ilikara <3435193369@qq.com>
---
 drivers/nvme/host/pci.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/nvme/host/pci.c b/drivers/nvme/host/pci.c
index 8ed5f1941f05..000004a24a92 100644
--- a/drivers/nvme/host/pci.c
+++ b/drivers/nvme/host/pci.c
@@ -3906,6 +3906,8 @@ static const struct pci_device_id nvme_id_table[] = {
 		.driver_data = NVME_QUIRK_NO_DEEPEST_PS, },
 	{ PCI_DEVICE(0x1e49, 0x0041),   /* ZHITAI TiPro7000 NVMe SSD */
 		.driver_data = NVME_QUIRK_NO_DEEPEST_PS, },
+	{ PCI_DEVICE(0x1fa0, 0x2283),   /* Wodposit NVMe SSD */
+		.driver_data = NVME_QUIRK_NO_SECONDARY_TEMP_THRESH, },
 	{ PCI_DEVICE(0x025e, 0xf1ac),   /* SOLIDIGM  P44 pro SSDPFKKW020X7  */
 		.driver_data = NVME_QUIRK_NO_DEEPEST_PS, },
 	{ PCI_DEVICE(0xc0a9, 0x540a),   /* Crucial P2 */
-- 
2.52.0

